### PR TITLE
Webside doc fixes

### PR DESCRIPTION
**Pull request by akartsky**
Website doc changes 

Deployed here for testing : https://akartsky.github.io/kubeflow-manifests/main

### CODE REVIEW

**Comment by rrrkharse** [comment]
Broken link: https://github.com/awslabs/kubeflow-manifests/blob/main/deployment/vanilla/kustomization.yaml#L57
Fix: https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/vanilla/kustomization.yaml#L57

In [Usage Tracking](https://akartsky.github.io/kubeflow-manifests/docs/about/usage-tracking/)

**Comment by rrrkharse** [comment]
Broken link: https://akartsky.github.io/kubeflow-manifests/docs/deployment/cognito/guide/kubeflow-manifests/docs/deployment/add-ons/load-balancer/guide#create-certificates-for-domain
Fix: Unsure

Text: If your cluster is not in N.Virginia(us-east-1), create an ACM certificate in us-east-1 for *.platform.example.com by following the process similar to [section 2.0](https://akartsky.github.io/kubeflow-manifests/docs/deployment/cognito/guide/kubeflow-manifests/docs/deployment/add-ons/load-balancer/guide/#create-certificates-for-domain)

**Comment by rrrkharse** [comment]
Broken link: https://github.com/awslabs/kubeflow-manifests/blob/main/docs/deployment/cognito/README-automated.md
Fix: https://github.com/awslabs/kubeflow-manifests/blob/main/docs/deployments/cognito/README-automated.md

Text: Take note of the following values from the previous step or awsconfigs/infra_configs/scripts/config.yaml if you used the [automated guide](https://github.com/awslabs/kubeflow-manifests/blob/main/docs/deployment/cognito/README-automated.md)

File: website/content/en/docs/deployment/cognito/guide-automated.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ This guide assumes you have Python 3.8 installed and that you have completed the
             NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
             istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
             ```
-        2. If `ADDRESS` is empty after a few minutes, check the logs of alb-ingress-controller by following [this guide](/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision)
+        2. If `ADDRESS` is empty after a few minutes, check the logs of alb-ingress-controller by following [this guide]({{< ref "/docs/troubleshooting-aws.md#alb-fails-to-provision" >}})
```
Comment (rrrkharse) [inline website/content/en/docs/deployment/cognito/guide-automated.md:85]: Need this same change [for LB guide](https://akartsky.github.io/kubeflow-manifests/main/docs/deployment/add-ons/load-balancer/guide/#update-the-domain-with-alb-address)

Text:
If ADDRESS is empty after a few minutes, check the logs of the controller by following the troubleshooting steps in [ALB fails to provision](https://awslabs.github.io/kubeflow-manifests/docs/troubleshooting-aws/#alb-fails-to-provision).

**Comment by rrrkharse** [comment]
In https://akartsky.github.io/kubeflow-manifests/main/docs/component-guides/kserve/#create-a-certificate

Broken link in:

Create an ACM certificate for *.platform.example.com and *.staging.platform.example.com in your cluster’s region by following the [create certificates for domain](https://akartsky.github.io/kubeflow-manifests/deployments/add-ons/load-balancer/guide/#create-certificates-for-domain) steps in the Load Balancer installation guide.

**Comment by rrrkharse** [comment]
In https://akartsky.github.io/kubeflow-manifests/main/docs/component-guides/profiles/#component-level-implementations

Broken link in:

You can find documentation about the AwsIamForServiceAccount plugin for specific components in the individual [component guides](https://akartsky.github.io/docs/component-guides/). Read on for general configuration instructions.

Fix: https://akartsky.github.io/kubeflow-manifests/main/docs/component-guides/

**Review by rrrkharse — CHANGES_REQUESTED** [review]

**Comment by surajkota** [comment]
@akartsky I recently merged #272 in which i followed this format for relative link. See this commit - https://github.com/awslabs/kubeflow-manifests/pull/272/commits/6387387b33e71bfb211e66a191c1253dedf3e3bc

Can you please rebase and check if it is correct?

**Review by rrrkharse — APPROVED** [review]

**Review by surajkota — CHANGES_REQUESTED** [review]
One broken link. Others we can find as we test after this is merged

File: website/content/en/docs/component-guides/kserve.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Use an ingress to set the [HTTP header conditions](https://docs.aws.amazon.com/e
 ### Create ingress
 
 1. Configure the following parameters for [ingress](https://github.com/awslabs/kubeflow-manifests/blob/main/awsconfigs/common/istio-ingress/overlays/api/params.env):
```
[the remark this answers is not in the document]
Comment (akartsky) [reply website/content/en/docs/component-guides/kserve.md:82]: yes 
let me fix this

File: website/content/en/docs/component-guides/kserve.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Once the certificate status changes to `Issued`, export the ARN of the certifica
 export certArn=<>
 ```
 
-If you are using Cognito for user authentication, see [Cognito](/kubeflow-manifests/docs/component-guides/kserve/#cognito-ingress). If you use Dex as the auth provider in your Kubeflow deployment, see [Dex](/kubeflow-manifests/docs/component-guides/kserve/#dex-ingress).
```
[the remark this answers is not in the document]
Comment (akartsky) [reply website/content/en/docs/component-guides/kserve.md:70]: This was fixed today morning 

https://akartsky.github.io/kubeflow-manifests/main/docs/component-guides/kserve/

File: website/content/en/docs/component-guides/kserve.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (akartsky) [reply website/content/en/docs/component-guides/kserve.md:82]: oh wait this is not even deployed to websites anymore 
so can't do it

**Review by surajkota — APPROVED** [review]

**Pull request merged**